src/components/PageSidebar/PageSidebar.test.tsx

```diff
@@ -1,20 +1,23 @@
-import React from 'react';
-import PageSidebar from './PageSidebar.component';
 import { render } from '@testing-library/react';
+import React from 'react';
 import { Router } from 'react-router-dom';
+import PageSidebar from './PageSidebar.component';
 import history from 'utils/misc/history';
 
 describe('PageSidebar', () => {
     it('should have link to candidates page', () => {
         const { getByLabelText } = render(
             <Router history={history}>
-                <PageSidebar location='http://localhost/candidates' match={{
+                <PageSidebar
+                  location='http://localhost/candidates'
```

Comment:
don't use localhost anywhere in testing. wee will create a config file later in development.

Reply:
Sure should replace this with some other url?